examples/example_patchextraction.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>please can you add an example to show that we can also extract a particular patch using an index for example, <code>patch[0]</code>  to extract patch at 0 location in dataframe.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Resolved in this commit https://github.com/TIA-Lab/tiatoolbox/pull/100/commits/bf66f2bedb92ada16fa7e5f51a39b420a32c8298